SignalServiceKit/src/Storage/TSDatabaseView.m

```diff
@@ -144,7 +144,7 @@ + (void)asyncRegisterUnseenDatabaseView
 
     [self registerMessageDatabaseViewWithName:TSUnseenDatabaseViewExtensionName
                                  viewGrouping:viewGrouping
-                                      version:@"1"
+                                      version:@"2"
```

Comment:
That was the other option - they'd have the same effect.  I don't see why we'd do both.  

I felt this approach was better so that we were consistent about the state of the `read` property and the `OWSReadTracking` protocol.  If we only ignored dynamic interactions, we might be surprised to find that some interactions in a thread still had `read` false when the "unseen" view was empty, etc.